src/main/java/org/datavyu/plugins/ffmpeg/MpvMediaPlayer.java

```diff
@@ -333,17 +311,23 @@ public int getImageHeight() {
 
         @Override
         public void onReady(PlayerStateEvent evt) {
-            container.setSize(getImageWidth(), getImageHeight());
+            if(masterClock != null) {
```

Comment:
Can we ensure that a masterClock is set?

Reply:
Actually, the master clock is not always set, we create the instance of the timer task only when an API client set an external clock to the players